pkg/providers/instancetype/suite_test.go

```diff
@@ -597,6 +597,32 @@ var _ = Describe("Instance Types", func() {
 		}
 		Expect(nodeNames.Len()).To(Equal(2))
 	})
+	It("should launch trn1 instances for AWS Neuron resource requests", func() {
+		nodeNames := sets.NewString()
+		provisioner.Spec.Requirements = []v1.NodeSelectorRequirement{
+			{
+				Key:      v1alpha1.LabelInstanceCategory,
```

Comment:
I would use a `node.kubernetes.io/instance-type` here and the specific `trn1.2xlarge` that is expected below. It will be less brittle if we add another trn instance type to the test data in the future.